skymp5-server/cpp/server_guest_lib/ActionListener.cpp

```diff
@@ -480,6 +480,20 @@ void ActionListener::OnChangeValues(const RawMessageData& rawMsgData,
     stamina = CropStaminaRegeneration(stamina, timeAfterRegeneration, actor);
   }
 
+  if (health != healthPercentage || magicka != magickaPercentage ||
+      stamina != staminaPercentage) {
+    std::string s;
+    s += Networking::MinPacketId;
+    s += nlohmann::json{
+      { "t", MsgType::ChangeValues },
+      { "data",
+        { { "health", health },
+          { "magicka", magicka },
+          { "stamina", stamina } } }
+    }.dump();
+    actor->SendToUser(s.data(), s.size(), true);
```

Comment:
Please unit test that the message is being delivered.

Here is an example on how to access messages sent during tests:
https://github.com/skyrim-multiplayer/skymp/blob/67bb639a2525e4bd193b9b704b86d430653a34c9/unit/PartOne_MovementTest.cpp#L30